godel_msgs/srv/TrajectoryExecution.srv

```diff
@@ -0,0 +1,13 @@
+# Joint trajectory
+trajectory_msgs/JointTrajectory trajectory
+
+# wait for execution to finish before returning
+bool wait_for_execution
+
+# Use the simulated robot or the actual robot
+bool simulate
+
+---
+
+# Return code
+int32 code
```

Comment:
I can't find any uses of said `code` in the software itself, so for now I'll get rid of it.

The code that uses this service checks for the return value of the service call directly. It might be nice to have enumerated failure types that could indicate failures such as `no keyence connection`. For failures that happen at execution time, there still questions though: Do we always return success immediately?

It seems that the right way to do it would be to implement some kind of action or action-esque interface that associates an execution with an ID number and broadcasts results (ID & result codes) when completed. This is probably out of the scope of the current system update.